masterlist.yaml

```diff
@@ -545,6 +563,33 @@ plugins:
       - 'More Exclusive Factions.ESP'
       - 'SwampFever.ESP'
 
+  - name: 'The_Vanilla_Quest_Tweaks_RP_Choices_Consequences_Super_Mega_Package_-_Ultimate_Edition.ESP'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/47466/'
+        name: 'The Vanilla Quest Tweaks RP Choices Consequences Super Mega Package - Ultimate Edition'
+    after: [ 'Religions Elaborated.ESP' ]
+    inc:
+      - 'A Cure for Vampirism - Skink''s Solution.ESP'
+      - 'Champion of Clutter.ESP'
+      - 'Duel of Honor - Improve the Chances.ESP'
+      - 'Fargoth in Distress.ESP'
+      - 'Fort Moonmoth Fundraiser Dinner.ESP'
+      - 'Harvest''s End Festival.ESP'
+      - 'Hentus Needs Pants Overhaul.ESP'
+      - 'Hiring Guards for the Redoran Stronghold - Honorable Solution.ESP'
+      - 'Immersive Neloth Reward.ESP'
+      - 'Libertarian Magical Services.ESP'
+      - 'Morrowind - Immanuel Kant Edition.ESP'
+      - 'Oath to Saint Roris Instead.ESP'
+      - 'Pacifist Options - When it Makes Sense.ESP'
+      - 'Redoran Founder''s Helm Add On- Honor the Ancestors.ESP'
+      - 'Strange Man at Gindrala Hleran''s House Overhaul.ESP'
+      - 'Teach Nels Llendo a Lesson.ESP'
+      - 'The Corpse and the Skooma Pipe Overhaul.ESP'
+      - 'The Frostmoth Smugglers - Properly Rewarded.ESP'
+      - 'Thelas'' Pillows Overhaul.ESP'
+      - 'Therana vs Trerayna.ESP'
```

Comment:
This should be sorted after `ROHT_2_0_8.ESP`.

Reply:
Like it is now?

Reply:
Yes. However, I'd get rid of the `Quest Overhauls` subsection & move `Main_Quest_Overhaul.ESP` back to where you had it. Subsections are usually for specific mods. If Morrowind's `Gameplay - Quests` section gets unwieldy at some point, something like `Gameplay - Quests - Overhauls` can be split off of it, but it's fine as-is at the moment.

Reply:
Done.